.changelog/5117.breaking.md

```diff
@@ -0,0 +1,4 @@
+go/tendermint: Change order of events returned from GetEvents()
```

Comment:
I guess under a strict definition this could be considered a breaking change as it modifies the order in API responses. However, this is not a consensus breaking change (no consensus issues in case some nodes apply this change)  and could be considered a bugfix. That would be useful if we want to backport this to `22.2.x` branch (the release process forbids breaking changes in point releases). 

Since oasis-indexer relies on this change, I would recommend backporting this to `stable/22.2.x` and making a new release, as otherwise the indexer backed nodes would need a custom-built version of oasis-node.

Reply:
Thanks!

I meanwhile already fixed the indexer so it does some heuristic reordering of events. So it does not depend on this change. I was hoping that once this PR goes in, I could remove the reordering on the indexer side, but I guess we'll also have to process archive data from Cobalt sooner or later, and I'm not sure it's worth backporting this change all the way there.

Still, it's nice to have flexibility. Good to know this is not consensus-breaking. Relabeling as bugfix.